lib/install.ps1

```diff
@@ -739,6 +739,7 @@ function link_current($versiondir) {
     }
 
     cmd /c mklink /j $currentdir $versiondir | out-null
+    attrib $currentdir +R /L
```

Comment:
And it also breaks the relink on updates! After an update the current directory still points to the old version!